wdl/imputation.wdl

```diff
@@ -144,7 +144,7 @@ task paste {
         File idx = outfile + ".tbi"
     }
     runtime {
-        docker: "gcr.io/finngen-refinery-dev/qc_imputation:test_paste_0.1.0"
+        docker: "eu.gcr.io/finngen-factory-staging/qc_imputation:test_paste_0.1.0"
```

Comment:
Does refinery service account have access to these images? If not, this will not work - anyway we have all images as inputs now in the update so you can remove this change to avoid merge conflicts